streams/src/test/java/org/apache/kafka/streams/processor/internals/ProcessorTopologyTest.java

```diff
@@ -337,39 +338,7 @@ public void testDrivingConnectedStateStoreTopology() {
         driver = new TopologyTestDriver(createConnectedStateStoreTopology("connectedStore"), props);
         final TestInputTopic<String, String> inputTopic = driver.createInputTopic(INPUT_TOPIC_1, STRING_SERIALIZER, STRING_SERIALIZER);
         final TestOutputTopic<Integer, String> outputTopic1 =
-            driver.createOutputTopic(OUTPUT_TOPIC_1, Serdes.Integer().deserializer(), Serdes.String().deserializer());
-
-        inputTopic.pipeInput("key1", "value1");
-        inputTopic.pipeInput("key2", "value2");
-        inputTopic.pipeInput("key3", "value3");
-        inputTopic.pipeInput("key1", "value4");
-        assertTrue(outputTopic1.isEmpty());
-
-        final KeyValueStore<String, String> store = driver.getKeyValueStore("connectedStore");
-        assertEquals("value4", store.get("key1"));
-        assertEquals("value2", store.get("key2"));
-        assertEquals("value3", store.get("key3"));
-        assertNull(store.get("key4"));
-    }
-
-    @Deprecated // testing old PAPI
-    @Test
-    public void testDrivingConnectedStateStoreInDifferentProcessorsTopologyWithOldAPI() {
```

Comment:
removed this (and some other tests) -- verified that we have equivalent tests for new Processor API for all of them